src/UKHO.ADDS.EFS.Domain/Files/Batch.cs

```diff
@@ -5,5 +5,6 @@ namespace UKHO.ADDS.EFS.Domain.Files
     public class Batch
     {
         public required BatchId BatchId { get; init; }
+        public DateTime ExchangeSetExpiryDateTime { get; set; } = DateTime.MinValue;  
```

Comment:
Can we rename it to BatchExpiryDateTime?
```suggestion
        public DateTime BatchExpiryDateTime { get; set; } = DateTime.MinValue;  
```

Reply:
Done.